setup.py

```diff
@@ -24,10 +24,11 @@
             'pyosf', 'requests[security]',
             'xlrd', 'openpyxl',
             'pyyaml', 'gevent', 'msgpack-python', 'psutil', 'tables'
+            'opencv-python',
             ]
 # some optional dependencies
 if platform == 'darwin':
-    required.append('pyobjc-core')
+    required.append('pyobjc-core', 'pyobjc-framework-Quartz')
```

Comment:
I was meaning submit a PR (I like the originator of fixes and features to get credit for their contributions). I'll pull the branch in and then you can do the fix in the main master (or I'm perfectly happy to if you prefer)